LICENCE

```diff
@@ -0,0 +1,18 @@
+Copyright (C) 2023 Crown Copyright (The National Archives)
```

Comment:
Did this mean to be part of this PR? looks like we already have a couple of licenses, 1 of which looks identical to this one (TNA one).

Need to delete the land registry one.

I'll do that in a separatre PR. Not sure why git is allowing multiple files with same path

Reply:
Resolved via #18